zellij-sessionizer.nu

```diff
@@ -6,25 +6,57 @@ def is-inside-zellij [] {
 
 # attach to a Zellij session by fuzzy finding projects
 #
-# the projects are computed by listing all the directories at depth between 1
-# and 2 recursively under the `path` argument.
+# # Examples
+#     opening a session by listing `nu-git-manager` repositories
+#     > zellij-sessionizer.nu (gm list --full-path)
+#
+#     open a session in local repos and documents
+#     > zellij-sessionizer.nu (
+#     >     [~/.local/share/repos/ ~/documents/] | each { ls $in | where type == dir | get name } | flatten
+#     > )
```

Comment:
Hi, Sorry for my absence!

isn't it the same as:

```sh
zellij-sessionizer $( fd . ~/Documents --type d) # Then it just gets piped into fzf?
```

Reply:
> Hi, Sorry for my absence!
> 
hey :wave: 
no worries :wink: 

> isn't it the same as:
> 
> ```shell
> zellij-sessionizer $( fd . ~/Documents --type d) # Then it just gets piped into fzf?
> ```
>
yeah it looks about the same :+1: 
i'm just using _pure_ Nushell commands but they are looking for depth-1 directories in all the directories in the list to the left :yum:

Reply:
Right!
So I'd probably add it as part of the alias (we can update the readme for it) instead of pushing it into the code as it feels like something some users would like where others would defer to an external app (such as `fd`)

WDYT?

Reply:
> Right! So I'd probably add it as part of the alias (we can update the readme for it) instead of pushing it into the code as it feels like something some users would like where others would defer to an external app (such as `fd`)
> 
> WDYT?
>
not sure i see what you mean here by "*add it as part of the alias instead of pushing it into the code*" :confused: 

and you're talking about the Bash implementation, right?

Reply:
>> add it as part of the alias instead of pushing it into the code
I mean that I think it should be part of the users config - like a parameter passed to their .rc aliases for zellij-sessionizer

I'm talking about both implementations as I try to maintain a[ single-responsibility philosophy](https://en.wikipedia.org/wiki/Single-responsibility_principle) as much as possible.

Reply:
ok i see, does that mean that you'd like the script not mentioning `find` or `fd` directly?

> i might miss something here :scream: :laughing:

Reply:
Yes!
give it some sane defaults and let the user decide about everything that's under the title of 'custumization'

Reply:
ooh yeah, i see what you mean now :relieved: 

i'd say, as `find` and `fd` were already there before this PR, we can leave this as-is for now and tackle the removal of these commands and making them configurable in another PR? :yum:

Reply:
So the solution I was thinking about is sort of a flat way of passing params into your find cmd, without explicitly mentioning it within the code.

Assuming you're using `fd` - I envision something like 

```sh
$ zellij_sessionizer [paths] ...rest_of_fds options
```

Then the script will just pass it as is.

Reply:
mm we can't do that really in Nushell right now :confused: